### DISCUSSION TITLE
How to always display top_bar instead of only displaying for a second?

### DISCUSSION THREAD

**Discussion by AKL55** [tomasklaen/uosc#439] [Q&A] [answered]
Already tried scripts similar to this
```ruby
mp.register_event('playback-restart', function()
  mp.command('script-message-to uosc flash-elements top_bar')
end)
```

**Comment by tomasklaen** [comment]
```
script-message-to uosc set-min-visibility top_bar 1
```

**Reply by AKL55** [reply]
Like this? it still showed the top bar for a second.i'm using the lastest uosc so he already flash the top bar on file change so maybe it's a conflict?
```ruby
mp.register_event('playback-restart', function()
  mp.command('script-message-to uosc set-min-visibility top_bar 1')
end)
```
What i want is the top_bar to be displayed permanently when i open mpv because i like to use uosc on my second monitor as a music player

**Comment by christoph-heinrich** [comment, marked as answer]
`top_bar_persistency=paused,audio,image,video`

**Reply by AKL55** [reply]
Thanks
